src/util/color/colorpalette.cpp

```diff
@@ -20,7 +20,8 @@ mixxx::RgbColor ColorPalette::colorForHotcueIndex(unsigned int hotcueIndex) cons
     int colorIndex;
     if (m_hotcueColorIndices.isEmpty()) {
         // For hotcue n, get nth color from palette
-        colorIndex = hotcueIndex;
+        // But use only  8 to avoid odd apearances on Hercules P32 or similar
+        colorIndex = hotcueIndex % 8;
```

Comment:
I did not mean that we should artificially limit palettes to 8 assigned hotcue colors. Users of controllers with >8 hotcues might want to set 16 distinct hotcue colors and I don't think we should prevent them from doing so. I meant that the default palette should only have 8 assigned hotcue colors.

Reply:
Fixed here: https://github.com/daschuer/mixxx/pull/50 
84a43ec